.github/workflows/gpu-tests.yml

```diff
@@ -74,14 +74,14 @@ jobs:
 
   nf-test-gpu:
     runs-on: "gpu"
-    name: "GPU | ${{ matrix.profile }} | ${{ matrix.shard }}"
+    name: "GPU Test | ${{ matrix.profile }} | ${{ matrix.shard }}"
     needs: nf-test-changes
     if: ${{ needs.nf-test-changes.outputs.modules != '[]' || needs.nf-test-changes.outputs.subworkflows != '[]' }}
     strategy:
       fail-fast: false
       matrix:
         shard: [1, 2]
-        profile: [docker_self_hosted, singularity] # conda?
```

Comment:
Please add conda. We should exclude the modules that don't use it afterwards, like we do in the normal tests